### DISCUSSION TITLE
Auto resistance on DKN Motion 1080

### DISCUSSION THREAD

**Discussion by rvanrooy** [cagnulein/qdomyos-zwift#368] [Q&A] [answered]
In Zwift, resistance is controlled in ERG mode on the DKN Motion 1080. but there is no resistance changes for gradient in free rides or events.  

Are there settings I should choose in Qdomyos-zwift or is this feature not available?

**Comment by cagnulein** [comment, marked as answer]
Did you disable the erg mode settings in qZ?

Il mar 15 giu 2021, 12:57 Reinhardt van Rooyen ***@***.***>
ha scritto:

**Reply by rvanrooy** [reply]
Wow :) I never thought of disabling it. Resistance does change with the incline with the setting disabled.

Is there a way to "change gears" by adding an offset dynamically? Admittedly, I've only played around for a few minutes, but there didn't seem to be controls in zwift or the companion app, and the buttons for resistance and target resistance in qZ work for a second, but then it reverts to the same level it was.


Awesome, thanks for the help thus far, and the great app

**Reply by cagnulein** [reply]
I can add this concept! I can add a new tile called "Gears" or "Virtual Gears" and you can add or sub an offset from the current resistance. What do you think? The best is to connect something like this https://zwifthacks.com/how-to-use-a-10-bluetooth-media-button-as-a-handlebar-mounted-zwift-game-controller/ to QZ directly and use them as gears!

**Reply by rvanrooy** [reply]
Would be pretty awesome if Qz had this concept built in, but I'll look into the controller option as well.

**Reply by cagnulein** [reply]
the controller have just 4 bluetooth buttons that QZ can intercept and act like a button on your phone. I'm saying this because i guess it's really hard to push a button on a touch screen when you're completely sweaty :)

**Reply by rvanrooy** [reply]
sure, so the button would integrate directly with QZ and not with the zwift client on the desktop. yeah, that would work nicely :)

**Reply by cagnulein** [reply]
yes exactly!

**Reply by martin-zh** [reply]
I would very much appreciate a "virtual gear" function. At the moment in Zwift it is often the case that I have to drive at very high speeds in order to be able to deliver the required power and thus be able to ride with the field. Being able to shift gears would help a lot....
BT buttons would be the killer, but for me 2 big (!) buttons in the app are enough. If buttons are used, universal ones should be used, which are available everywhere.
